modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -125,8 +115,8 @@ Two tables are created per object store:
 
 * MuleSoft recommends that you create a dedicated database/schema that will only be used for JDBC store.
 * The database username needs to have permission to:
-** Create objects in the database, which means DDL CREATE, DROP for tables.
-** DML permissions on the objects it creates (INSERT, UPDATE, DELETE, SELECT)
+** Create objects in the database, which means `DDL CREATE`, `DROP` for tables.
+** DML permissions on the objects it creates (`INSERT`, `UPDATE`, `DELETE`, `SELECT`)
```

Comment:
This list item does not grammatically follow its fragment introduction: "permission to" do something.
Does this work?
```suggestion
** Access and manage the objects it creates (`INSERT`, `UPDATE`, `DELETE`, and `SELECT`)
```